pkg/tempopb/tempo.proto

```diff
@@ -42,13 +42,15 @@ message TraceByIDMetrics {
 
 // SearchRequest takes no block parameters and implies a "recent traces" search
 message SearchRequest {
+  // TraceQL query
```

Comment:
changing proto field #s is backwards incompatible. just add the query to the end as field 8

Reply:
Right, I wasn't sure but forgot to check.